atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -103,6 +123,7 @@ def __init__(self, tasker, workflow, kojihub, url,
                                   namespace=self.namespace)
         self.osbs = OSBS(osbs_conf, osbs_conf)
         self.build_id = None
+        self.nvr_image = None
```

Comment:
I bundled three fixes for warnings together in one commit. Only one of the warnings is new with this PR.